resources/config/defaults/boards/276.txt

```diff
@@ -2,7 +2,7 @@ STAGE 276
 ROW_1 Air,Wood,Air,Wood,Air,Wood
 FROW_1 false,false,false,false,false,false
 ROW_2 Air,Wood,Air,Wood,Air,Wood
-FROW_2 false,false,false,false,false,false
+FROW_3 false,false,false,false,false,false
```

Comment:
Don't worry about it, I've copied the verified parts from your pull request into what I was already working on. I'll be pushing it in either tonight or tomorrow and begin the testing phase (and hopefully the deployment phase shortly after).